assets/dev/scss/frontend/_container.scss

```diff
@@ -137,18 +137,17 @@
 		// Since the default is `min-width: auto` for flex items.
 		min-width: 0;
 
-		// Fix iframe-based widgets (ED-5338).
-		// TODO: Find a better solution.
+		// Fix iframe-based widgets (ED-5338) & widget with empty placeholder.
 		&-video,
-		&-google_maps {
-			flex-basis: 100%;
+		&-google_maps,
+		&-empty {
+			width: 100%;
 		}
 
 		// Fix swiper-based widgets broken (ED-6849).
 		// TODO: Use `:has( .swiper-container )` selector when available && remove this class from `frontend/utils/swiper.js`.
 		&.e-widget-swiper {
-			max-width: 100%;
-			flex-basis: 100%;
+			width: 100%;
```

Comment:
Doesn't this cause side effects?
I feel like setting the width to 100% can clash with other settings related to widget width.
How does this work with the Custom Width control in `Advanced > Layout > Width (set to Custom)`?